### DISCUSSION TITLE
Github Tag Bug

### DISCUSSION THREAD

**Discussion by Lucaffo** [community/community#4557] [Other Feature Feedback, Questions, & Ideas]
Hello! today I was caught by this bug that is starting to bother me a little.

When you open your tag list on github:

![image](https://user-images.githubusercontent.com/55745404/124930668-d2228980-e001-11eb-994b-f41a49714c08.png)

You have these 3 dots on side that you can click and overview the tag description:

![image](https://user-images.githubusercontent.com/55745404/124930774-e8304a00-e001-11eb-9f3a-703b8f4af21a.png)

Until now no problem, but if you click on the tag for a complete description (Because sometimes you can't overview the description if it's too large), you encounter this "Bug" at least i don't know how to name it.

![image](https://user-images.githubusercontent.com/55745404/124931149-39d8d480-e002-11eb-872d-81e2b5bcc6e6.png)

Github puts the first line of the description inside the tag title.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
